apps/condo/domains/ticket/schema/TicketShareService.test.js

```diff
@@ -0,0 +1,59 @@
+const { NEW_OR_REOPENED_STATUS_TYPE } = require('@condo/domains/ticket/constants')
+const { makeLoggedInClient } = require('@condo/domains/user/utils/testSchema')
+const { createTestUser } = require('@condo/domains/user/utils/testSchema')
+const { makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+const { v4: uuid } = require('uuid')
+const { createTestTicket } = require('@condo/domains/ticket/utils/testSchema')
+const { TICKET_SHARE_MUTATION } = require('@condo/domains/ticket/gql')
+const { makeClient } = require('@core/keystone/test.utils')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+
+describe('TicketShareService', () => {
+    describe('User', () => {
+        it('can share ticked with selected organization', async () => {
+            const client = await makeClientWithProperty()
+            const [ticket] = await createTestTicket(client, client.organization, client.property)
+            const res = await client.query(TICKET_SHARE_MUTATION, {
+                data: { sender: client.userAttrs.sender, users: [client.id], ticketId: ticket.id },
+            })
+            console.log('data[0].id', ticket.id)
+            console.log('client', client.userAttrs.sender)
+            // console.log('ticket', data)
+            console.log('ticketShare', res)
+            expect(true).toBeTruthy()
+
+            // expect(data).toBeInstanceOf(Array)
+            // expect(data.length).toBeGreaterThanOrEqual(1)
+            // const clientTicket = data.find(e => e.statusType === NEW_OR_REOPENED_STATUS_TYPE)
+            // expect(clientTicket).toBeDefined()
+            // expect(clientTicket.currentValue).toEqual(1)
+            // expect(clientTicket.growth).toEqual(0)
+        })
+
+        // it('can not share ticked with another organization', async () => {
```

Comment:
add tests for such cases:
 1) anonymous user
 2) not related to this organization user
 3) related to this organization user
...